resources/lib/ipwww_common.py

```diff
@@ -247,55 +247,46 @@ def InitialiseCookieJar():
 
 
 def SignInBBCiD():
-    sign_in_url="https://account.bbc.com/signin"
-
-    username=ADDON.getSetting('bbc_id_username')
-    password=ADDON.getSetting('bbc_id_password')
-
-    post_data={
-               'username': username,
-               'password': password,
-               'attempts':'0'}
-    
-    #Regular expression to get 'nonce' from login page
-    p = re.compile('action="([^""]*)"')
-    
     with requests.Session() as s:
-        resp = s.get('https://www.bbc.com/', headers=headers)
-
-        # Call the login page to get a 'nonce' for actual login
-        signInUrl = 'https://session.bbc.com/session'
-        resp = s.get(signInUrl, headers=headers)
-        m = p.search(resp.text)
-        url = m.group(1)
-
-        url = "https://account.bbc.com%s" % unescape(url)
-        resp = s.post(url, data=post_data, headers=headers)
+        s.headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:110.0) Gecko/20100101 Firefox/110.0'}
```

Comment:
Use the global variable user_agent instead of hard coding FF110, please. Background: I keep updating the user agent every now and then, and it should be consistent. Maintaining in one place is easier.

Reply:
Yes, you're right of course. Change is on its way.